src/main/java/edu/kis/powp/jobs2d/drivers/command/SetPositionCommand.java

```diff
@@ -0,0 +1,14 @@
+package edu.kis.powp.jobs2d.drivers.command;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+
+public class SetPositionCommand extends AbstractDriverCommand{
+    public SetPositionCommand(int x, int y) {
+        super(x, y);
+    }
+
+    @Override
+    public void execute(Job2dDriver driver) {
```

Comment:
Original code used tabs instead of spaces. It is important to keep the original convention. It makes tracking actual changes more difficult.